client/app/components/packages/scope-of-work-draft/edit.js

```diff
@@ -0,0 +1,24 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+
+export default class PackagesScopeOfWorkDraftEditComponent extends Component {
```

Comment:
The folder naming needs to be consistent. Please change this file to `draft-scope-of-work/edit.js` and the Component to PackagesDraftScopeOfWorkEditComponent

Reply:
@godfreyyeung Let's check with @allthesignals first:)

Reply:
Okay, Just had a conversation with @allthesignals . You're good to go!
We're planning on doing some extra reorganizing later to use folders to dilineate draft vs final, etc.
e.g. `sow/draft.hbs` and `sow/final.hbs`, etc.